scripts/env.sh

```diff
@@ -0,0 +1,25 @@
+# for mongodbtoolchain python3
+export PATH=/opt/mongodbtoolchain/v3/bin:$PATH
+
+# Check if not called with -g or --run-global flag
+if [[ "$*" != *-g* ]]; then
+    if [[ ! -e "$SCRIPTS_DIR/venv/.setup-complete" ]]; then
+        echo "Installing virtual environment."
+        echo "To run without a virtual environment, use the -g or --run-global option."
+        echo "creating new env with python: $(command -v python3)"
+        pushd "$SCRIPTS_DIR" >/dev/null
+            python3 -m venv venv
+        popd >/dev/null
+    fi
+
+    set +u
+        source "${SCRIPTS_DIR}/venv/bin/activate"
+    set -u
+
+    if [[ ! -e "$SCRIPTS_DIR/venv/.setup-complete" ]]; then
```

Comment:
you've already done the `.setup-complete` check so no need to do it again 💅

Reply:
I really hate the way this logic is expressed, but the second half of the logic finishes the setup after the venv is sourced.

I think it might be clearer to just copy and paste the venv activation into the setup block and early return.